packages/mui-base/src/Popper/Popper.tsx

```diff
@@ -233,7 +231,7 @@ const PopperTooltip = React.forwardRef(function PopperTooltip<
   return (
     <Root {...rootProps}>{typeof children === 'function' ? children(childProps) : children}</Root>
   );
-}) as OverridableComponent<PopperTooltipTypeMap>;
+}) as PolymorphicComponent<PopperTooltipTypeMap>;
```

Comment:
@michaldudak this change seems to make `yarn proptypes` change the proptypes of `Menu.tsx` and `Select.tsx` (both use `Popper` as default `Root` component) like in the picture. Any idea what might causes this?
![image](https://user-images.githubusercontent.com/56389521/234892722-725110c9-1c62-4e20-b81d-2e2e0d295755.png)

Reply:
You can add `/* @typescript-to-proptypes-ignore */` before running `yarn proptypes` to prevent that change

```
  container: PropTypes /* @typescript-to-proptypes-ignore */.oneOfType([
    HTMLElementType,
    PropTypes.func,
  ]),
```

Reply:
Thank you!

Reply:
In fact, ignore my comment above, that's not how we should handle it

Reply:
have you found another way to deal with the problem?